tests/oss-fuzz/build.sh

```diff
@@ -42,15 +42,15 @@ export ORIG_CXXFLAGS="$CXXFLAGS"
 export CFLAGS=""
 export CXXFLAGS=""
 
-# fuzz flags are problematic with meson (hence no dav1d) and no point in fuzzing fuzztest.
-cd ext && bash dav1d.cmd && bash fuzztest.cmd && cd ..
+# Use normal flags for projects built outside of libavif (e.g. dav1d with meson is
```

Comment:
"projects built outside of libavif" also include libaom, but aom.cmd is not in this list (line 47).

Should we move aom.cmd to this list?

Reply:
Yes and no. I reduced the list to only have dav1d which truly does not work here with sanitizer flags.
We could add the ones we build outside (like aom, sharpyuv,libyuv) so that the fuzzing info is not present (for speed or so that they do not pollute the coverage) but we are deeply involved with those projects so we might as well put them.